iothub/device/src/IotHubBaseClient.cs

```diff
@@ -483,6 +450,8 @@ internal void OnDesiredStatePatchReceived(TwinCollection patch)
             _ = _desiredPropertyUpdateCallback.Invoke(patch, _twinPatchCallbackContext);
         }
 
+        private protected abstract void AddToPipelineContext();
```

Comment:
+1

Reply:
This method will disappear shortly when we consolidate the setMessageReceiveHandler code, FYI